MapboxVision/src/main/java/com/mapbox/vision/utils/system/SystemTime.kt

```diff
@@ -0,0 +1,11 @@
+package com.mapbox.vision.utils.system
+
+interface SystemTime {
```

Comment:
Plain `System`, not `SystemTime`?

Reply:
System already exist and has pretty huge API, I suggest to use small interface for Time only

Reply:
Well, we might implement only those methods from System, that we actually need. 

What about renaming class to `interface Time { fun millis() }`  ?
Usage will become shorter.

Reply:
done